cmd/bugzilla-backporter/main.go

```diff
@@ -102,6 +108,36 @@ func gatherOptions() (options, error) {
 	return o, nil
 }
 
+func getAllTargetVersions(configFile string) (sets.String, error) {
+	// Get the versions from the plugins.yaml file to populate the Target Versions dropdown
+	// for the CreateClone functionality
+	b, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		return nil, err
+	}
+	np := &plugins.Configuration{}
+	if err := yaml.Unmarshal(b, np); err != nil {
+		return nil, err
```

Comment:
Nit: Can you wrap errors rather then directly returning them so debugging becomes easier? Golang doesn't provide stracktraces, and using the wrapping helps to easely figure out where exactly an error happend.
Sample:
```
return nil, fmt.Errorf("failed to unmarshal yaml: %w")
```

Reply:
Instead of wrapping with failed to umarshal would it be better to just put in the function name where it occurred if the objective is to figure out exactly where the error happened?

Reply:
That is essentially what we do but we do it in the caller, not in the callee, otherwise you have to change all errors if you decide to move code from one function in another. Additionally its simpler, because if someone wrote wrapping like 
`failed to do http requst` here you will directly see that this is wrong whereas if the wrapping referred to the function it is in you need to remember the function name which is declared further up.